src/util/kafka/StreamHandler.cpp

```diff
@@ -87,19 +94,21 @@ void StreamHandler::listen_to_kafka_topic() {
         obj["source"] = sourceJson;
         obj["destination"] = destinationJson;
         obj["properties"] = edgeJson["properties"];
-        long temp_s = partitionedEdge[0].second;
-        long temp_d = partitionedEdge[1].second;
+        long part_s = partitionedEdge[0].second;
+        long part_d = partitionedEdge[1].second;
+        long temp_s = part_s % atoi((Utils::getJasmineGraphProperty("org.jasminegraph.server.nworkers")).c_str());
```

Comment:
Shall we first assign `atoi((Utils::getJasmineGraphProperty("org.jasminegraph.server.nworkers")).c_str())` to a local variable and then use the local variable in the lines 99 and 100? That way we can avoid unnecessary cost of reading the config value.

Reply:
Done.

Reply:
Ok